src/ui/plot.jsx

```diff
@@ -5,9 +5,11 @@ import { sum } from "lodash";
 
 import {PhysicsContext} from '../state'
 
+import { Num } from '.'
+
 const evBins = new Float64Array(1000).map((v, i) => i * 0.01 + 0.005);
 
-export function NuSpectrumPlot({ cores, spectrum, detector}) {
+export function NuSpectrumPlot({ cores, spectrum, detector, reactorLF}) {
```

Comment:
A new input was added to this function, take a look at how it is "called":
https://github.com/geoneutrinos/reactors/blob/1a21a993571349d9cb7fe604a9a03b56ebb61bf2/src/App.js#L190-L194

Reply:
${} inserts variable to string

but variable not found so site crashes... helpful hint directs problem to /src/App.js

need to add "reactorLF={reactorLF}" between lines 193 and 194

so this commit will crash site until App.js is modified

not sure if changes must be done in tandem or if serial (App.js then this one) is OK